### PR TITLE
style: enforce buildifier lint rules

### PR DESCRIPTION
**Pull request by manekinekko**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by manekinekko** [comment]
requesting reviews from @alexeagle and @gregmagolan.

**Review by gregmagolan — APPROVED** [review]
LGTM

**Comment by alexeagle** [comment]
Caretaker: this is a cherry-pick to patch branch of a commit already on master. No need for re-approve or google3 sync

**Review by filipesilva — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
merged as 9226b42

File: bazel-out [hunk 1]
```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_wassim/b60489123ecff60262797b9b261f5599/execroot/angular/bazel-out
```
Comment (devversion) [inline bazel-out]: For some reason this sneaked into the `7.2.x` branch now 😄 FYI: I will remove this into my upcoming patch-only PR

**Pull request closed without merging**